similarity/config/configurations.properties

```diff
@@ -0,0 +1,20 @@
+wordMetric	paragram
```

Comment:
SimConfigurator already have everything. We could drop this file. I just think it's easier for user to change configurations file than changing SimConfigurator class.